contracts/src/CrossChainApplications/examples/NativeTokenBridge/NativeTokenDestination.sol

```diff
@@ -151,7 +224,8 @@ contract NativeTokenDestination is TeleporterOwnerUpgradeable, INativeTokenDesti
             );
         }
 
-        uint256 totalBurnedTxFees = BURNED_TX_FEES_ADDRESS.balance;
+        uint256 scaledAmount = _scaleTokens(BURNED_TX_FEES_ADDRESS.balance, false);
```

Comment:
should we check if `scaledAmount` is zero here too?

Reply:
We're killing this function in another PR, so I'm not going to worry about it here